robotmap.py

```diff
@@ -6,7 +6,8 @@
 RIGHT_LEADER_ID = 7
 RIGHT_FOLLOWER_ID = 9
 
-COLOR_SENSOR_MOTOR = 3
+COLOR_SENSOR_MOTOR = 11
```

Comment:
Last I ask electrical, COLOR_SENSOR_MOTOR was at port 3
Ask electrical to see which the ports are for Shooter and color sensor motor